moveit_plugins/moveit_ros_control_interface/README.md

```diff
@@ -22,12 +22,12 @@ In your MoveIt! launch file (e.g. `ROBOT_moveit_config/launch/ROBOT_moveit_contr
 <arg name="moveit_controller_manager" default="moveit_ros_control_interface::MoveItControllerManager" />
 ```
 
-And make sure so set the `ros_control_namespace` parameter to the namespace (without the /contoller_manager/ part) of the ros_control-based node you like to interface.
-If you are using the `moveit_setup_assistent` you can add it to `ROBOT_moveit_config/config/ROBOT_controllers.yaml`, e.g.:
+And make sure to set the `ros_control_namespace` parameter to the namespace (without the /controller_manager/ part) of the ros_control-based node you like to interface.
+If you are using the `moveit_setup_assistent` you can add it to `ROBOT_moveit_config/config/controllers.yaml`, e.g.:
```

Comment:
The MSA does not create a `controllers.yaml` at all.
[This tutorial](http://docs.ros.org/kinetic/api/moveit_tutorials/html/doc/pr2_tutorials/planning/src/doc/controller_configuration.html) explains that they should create and populate a file called `controllers.yaml` manually.

I'd prefer to change this.